yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -488,6 +496,10 @@ export const PXESchema: ApiSchemaFor<PXE> = {
     .function()
     .args(schemas.AztecAddress, schemas.Fr, schemas.Fr)
     .returns(z.tuple([schemas.BigInt, SiblingPath.schemaFor(L1_TO_L2_MSG_TREE_HEIGHT)])),
+  getL2ToL1MembershipWitness: z
+    .function()
+    .args(z.number(), schemas.Fr)
+    .returns(z.tuple([schemas.BigInt, SiblingPath.schema])),
```

Comment:
If it wasn't disabled, the circuit-types test should fail because it checks that _every function_ in the schema is exercised.

Reply:
Fixed